packaging/os/zypper_repository.py

```diff
@@ -58,16 +58,30 @@
         required: false
         default: "no"
         choices: [ "yes", "no" ]
-        aliases: []
     refresh:
         description:
             - Enable autorefresh of the repository.
         required: false
         default: "yes"
         choices: [ "yes", "no" ]
-        aliases: []
-notes: []
-requirements: [ zypper ]
+    priority:
+        description:
+            - Set priority of repository. Packages will always be installed
+              from the repository with the smallest priority number.
+        required: false
+        version_added: "2.1"
+    overwrite_multiple:
+        description:
+            - Overwrite multiple repository entries, if repositories with both name and
+              URL already exist.
+        required: false
+        default: "no"
+        choices: [ "yes", "no" ]
+        version_added: "2.1"
+requirements: 
+    - "zypper  >= 1.0"
+    - "openSuSE >= 11.1"
```

Comment:
opensuse or sles are not really requirements. That should be hints.
- "zypper >= 1.0" # included in openSuSE >= 11.1 or SuSE Linux Enterprise Server/Desktop >= 11.0